nuitka/nodes/ExceptionNodes.py

```diff
@@ -334,6 +334,33 @@ def __init__(self, source_ref):
     def computeExpressionRaw(self, trace_collection):
         return self, None, None
 
+class ExpressionCaughtExceptionGroupMatch(ExpressionBase):
+    kind = "EXPRESSION_CAUGHT_EXCEPTION_GROUP_MATCH"
+
+    __slots__ = ("caught", "catching")
+
+    def __init__(self, caught, catching, source_ref):
+        ExpressionBase.__init__(self, source_ref)
+        self.caught = caught
```

Comment:
The way you announce children to Nuitka, is wrong here, that's probably all your remaining issues.